Makefile

```
@@ -33,7 +33,7 @@ $(LOCAL_BINARY): $(SOURCES)
 
 .PHONY: test
 test:
-	env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT go test -timeout=120s -v -cover ./ecs-cli/modules/...
+	env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT GOCACHE=$$HOME/.gocache go test -timeout=120s -v -cover ./ecs-cli/modules/...
```

hi @dmnlk! Thanks for the change, could you modify this to the following?
```suggestion
	env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT GOCACHE=$$GOCACHE go test -timeout=120s -v -cover ./ecs-cli/modules/...
```
So that each developer can decide where their build cache is.

---

thanks!

---

I accepted your suggested changes, but squash commit is better?

---

yeah, I agree squashed commits would be better. I can look into merging your branch with the fast-forward option (as [outlined here](https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits)) once the Travis build passes.